packages/apps-config/src/api/spec/ferrum.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017-2023 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { OverrideBundleDefinition } from '@polkadot/types/types';
+
+// structs need to be in order
+/* eslint-disable sort-keys */
+
+const definitions: OverrideBundleDefinition = {
+  types: [
+    {
+      // on all versions
+      minmax: [0, undefined],
+      types: {
+        AccountId: 'EthereumAccountId',
+        AccountId20: 'EthereumAccountId',
+        AccountId32: 'EthereumAccountId',
```

Comment:
This is probably not quite correct - `EthereumAccountId` represents a `[u8; 20]` internally, while `AccountId32` represents an `[u8; 32]`. 

So the types are mismatched on this one and anything that uses it (some ETH-compatible chains for instance has a bridge to other Substrate-chains) will fail horribly.

The first 2 is all that is needed to map to ETH-compatible.